calblueprint/siren-web
# Bry/linking intake dash

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Linking the intake dashboard to the client view
(!!!WIP!!!)

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] wrapped rows with Link component
 - [x] display the navigation bar on top
 - [x] pass in client info
 - [x] correctly display client info into client view
 - [x] styled <3

## How can the reviewer test your code? 👩‍💻
_example: run `npm run dev` and click on a table row. This should direct to client view

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 
Currently stuck on passing Object types into router.query


## Resources 📔
_link online resources and related PRs_ 
https://nextjs.org/learn/basics/navigate-between-pages/client-side

<img width="1292" alt="Screen Shot 2022-04-23 at 12 13 06 AM" src="https://user-images.githubusercontent.com/60284767/164884403-ce71f4aa-3f65-438c-9f88-b5b5163c90f1.png">
<img width="1294" alt="Screen Shot 2022-04-23 at 12 14 07 AM" src="https://user-images.githubusercontent.com/60284767/164884418-da7f0a63-9905-4d36-88be-7b43d8e4e968.png">
<img width="1275" alt="Screen Shot 2022-04-23 at 12 14 18 AM" src="https://user-images.githubusercontent.com/60284767/164884424-52b330f8-ce82-4b23-9e64-849c3520773d.png">
<img width="1291" alt="Screen Shot 2022-04-23 at 12 14 50 AM" src="https://user-images.githubusercontent.com/60284767/164884440-73645543-198a-477e-89a3-027992b2e8a0.png">
<img width="1297" alt="Screen Shot 2022-04-23 at 12 15 02 AM" src="https://user-images.githubusercontent.com/60284767/164884446-6ecb5a32-992f-4fed-87d2-df85de98cd19.png">




🧜‍♀️ cc: @shannonbonet @phoebeli23
